package.json

```diff
@@ -104,9 +103,19 @@
     "e2e/*"
   ],
   "dependencies": {
+    "@angular/animations": "~13.1.0",
+    "@angular/common": "~13.1.0",
+    "@angular/compiler": "~13.1.0",
+    "@angular/core": "~13.1.0",
+    "@angular/forms": "~13.1.0",
+    "@angular/platform-browser": "~13.1.0",
+    "@angular/platform-browser-dynamic": "~13.1.0",
+    "@angular/router": "~13.1.0",
```

Comment:
Can these packages go in the workspace's `package.json`? Just to keep the root deps clean?